<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LungNoduleROI</class>
 <widget class="qMRMLWidget" name="LungNoduleROI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>913</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignVCenter">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Nodule ROI</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ctkCollapsibleButton" name="CollapsibleButton_3">
        <property name="text">
         <string>Single Case</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="7">
            <widget class="QPushButton" name="centroidManualButton">
             <property name="text">
              <string>Enter Centroid Manually</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="roiSizeLabelStatic">
             <property name="text">
              <string>ROI Size:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="5">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Output Volume Name</string>
             </property>
            </widget>
           </item>
           <item row="8" column="6" colspan="2">
            <widget class="QLineEdit" name="fileName">
             <property name="text">
              <string>Output ROI</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3" colspan="4">
            <widget class="QSlider" name="roiSizeSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Volume:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="6">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="sLineEdit">
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>S</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cLineEdit">
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>C</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="aLineEdit">
               <property name="maximumSize">
                <size>
                 <width>45</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="7">
            <widget class="QPushButton" name="noduleCentroidButton">
             <property name="text">
              <string>Cursor Placement</string>
             </property>
            </widget>
           </item>
           <item row="6" column="5" colspan="3">
            <widget class="ctkCollapsibleButton" name="CollapsibleButton_5">
             <property name="text">
              <string>More Options</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="interpolationCheckBox">
                <property name="text">
                 <string>Slicer Interpolation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="roiCheckBox">
                <property name="text">
                 <string>Non-Isotropic ROI Size</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="nonIsoGridLayout">
                <item row="0" column="3">
                 <widget class="QLineEdit" name="sLineEditNonIso">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="aLabelNonIso">
                  <property name="text">
                   <string>Axial</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLineEdit" name="cLineEditNonIso">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="sLabelNonIso">
                  <property name="text">
                   <string>Saggital</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="cLabelNonIso">
                  <property name="text">
                   <string>Coronal</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QSlider" name="cSliderNonIso">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="3">
                 <widget class="QLineEdit" name="aLineEditNonIso">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QSlider" name="aSliderNonIso">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QSlider" name="sSliderNonIso">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="11" column="5" colspan="3">
            <widget class="QPushButton" name="applyButton">
             <property name="text">
              <string>Create Single ROI</string>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QLineEdit" name="roiSizeLabel">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3" colspan="5">
            <widget class="qMRMLNodeComboBox" name="volumeComboBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="nodeTypes">
              <stringlist>
               <string>vtkMRMLScalarVolumeNode</string>
              </stringlist>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleButton" name="CollapsibleButton_4">
        <property name="text">
         <string>Batch Processing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="4" column="4" colspan="2">
            <widget class="QPushButton" name="applyButton_2">
             <property name="text">
              <string>Create ROI Batch</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Path to Volumes</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Path to Centroids &amp; Sizes</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Output Path</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="text">
      <string>Nodule Segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="qMRMLSegmentEditorWidget" name="SegmentEditorWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
